src/library/scala/collection/LowPriority.scala

```diff
@@ -0,0 +1,11 @@
+package scala.collection
+
+class LowPriority {
```

Comment:
Most (all?) package object contents including this `LowPriority` should be removed. They were used for implicit conversions in `strawman.collection` but are no longer necessary in `scala.collection` where `Predef` is used instead.